.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Bug report
```

Comment:
Nice naming

Reply:
Thanks

Reply:
Do you think people will notice that we only write this to test the webhook?

Reply:
Just another test